### ISSUE TITLE
[Ivy] create embedded view is not added

### ISSUE THREAD

**Issue by lujian98** [angular/angular#37054]
# 🐞 bug report

### Affected Package
@angular/core

### Is this a regression?

Yes, worked perfectly fine in v8 without Ivy. Works also without Ivy in v9.
It's interesting that in Stackblitz it works as intended, but when using Chrome/FireFox and ng serve locally it doesn't work.

### Description

I try to create embedded view from template and then add the embedded view to the Content Children. However, the embedded views are not added when Ivy is on with v9.

<pre><code>
  @ViewChild("templateToAppend", {static: true}) templateToAppend: TemplateRef<any>;
  @ContentChildren(SomeOtherComponent, { descendants: true, read: ViewContainerRef }) someOtherComponents: QueryList<ViewContainerRef>;

  ngAfterContentInit() {
    this.someOtherComponents.forEach(ap => ap.createEmbeddedView(this.templateToAppend));
  }
</code></pre>

## 🔬 Minimal Reproduction
https://stackblitz.com/edit/github-f2y5an?file=src%2Fapp%2Fexample%2Fexample.component.ts

https://github.com/lujian98/Angular-create-embeded-view

## 🔥 Exception or Error
<pre><code>

</code></pre>

## 🌍  Your Environment

**Angular Version:**
<pre><code>

Angular CLI: 9.1.5
Node: 10.15.3
OS: linux x64

Angular: 9.1.6
... animations, common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... router
Ivy Workspace: Yes

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.901.5
@angular-devkit/build-angular     0.901.5
@angular-devkit/build-optimizer   0.901.5
@angular-devkit/build-webpack     0.901.5
@angular-devkit/core              9.1.5
@angular-devkit/schematics        9.1.5
@angular/cli                      9.1.5
@ngtools/webpack                  9.1.5
@schematics/angular               9.1.5
@schematics/update                0.901.5
rxjs                              6.5.5
typescript                        3.7.5
webpack                           4.42.0

</code></pre>

**Anything else relevant?**

**Comment by pkozlowski-opensource** [comment]
@lujian98 I've simplified a bit your scenario here: https://stackblitz.com/edit/github-f2y5an-hmzlxt?file=src%2Fapp%2Fapp.component.ts

Still, I'm confused about the exact problem. You are saying:

> It's interesting that in Stackblitz it works as intended, but when using Chrome/FireFox and ng serve locally it doesn't work.
> However, the embedded views are not added when Ivy is on with v9.

Could you please be more specific? When you say "views are not added" - are they missing from the DOM (not displayed) or something else is going on? 

I'm afraid that we need more info here...

**Comment by lujian98** [comment]
![angular-9-ivy-on](https://user-images.githubusercontent.com/45267937/81885041-15e54480-955f-11ea-97f1-3f4508e9010f.png)

![angular-9-ivy-off](https://user-images.githubusercontent.com/45267937/81885049-1b428f00-955f-11ea-8fc3-663d202a8179.png)

**Comment by lujian98** [comment]
The embedded views are missing from the DOM when Ivy is on with "enableIvy": true.

**Comment by lujian98** [comment]
![embedded-views-missing-with-ivy-on](https://user-images.githubusercontent.com/45267937/81885508-14684c00-9560-11ea-8f5c-afd462b0a153.png)
![embdded-views-with-ivy-off](https://user-images.githubusercontent.com/45267937/81885510-16320f80-9560-11ea-833c-984708c9fada.png)

**Comment by pkozlowski-opensource** [comment]
@lujian98 I find it surprising that stackblitz works fine but your application don't. Could you please do the following:
* make sure that your app uses the same (recent) version of Angular as in stackblitz;
* try to simplify your repo to the absolute minimum (as I did in https://stackblitz.com/edit/github-f2y5an-hmzlxt?file=src%2Fapp%2Fapp.component.ts) and see if you can still reproduce
* provide us with the _exact_ command you use to run your app

Thnx!

**Comment by dmitry-kostin** [comment]
I have a similar problem, but I have problem only with ssr rendering.
With following construction:

> constructor(
>         private _vcr: ViewContainerRef,
>     )

> this._vcr.createEmbeddedView(this.template)

It should create view on the same level as current component is, but it goes deeper like it doesn't see parent view;
Here is screen to compare 
![image](https://user-images.githubusercontent.com/1920678/82078717-4ad1d280-96ea-11ea-9d7b-3c1b13d30f78.png)
on the right: after ssr, with javascript turned off
on the left: after browser re-render
So basically i have flat structure instead of nested where every node have its own view
ps not sure I'm able reproduce this

**Comment by pkozlowski-opensource** [comment]
I'm going to close this one as we didn't get a reproduce scenario. Please open a new issue attaching a reproduce scenario that we could use to see the bug.
